src/components/App/AppHead.tsx

```diff
@@ -272,7 +272,7 @@ export const AppHead = (props: IProps) => {
           {(isOffline || orbitStatus !== undefined || !connected) && (
             <CloudOffIcon className={classes.spacing} color="action" />
           )}
-          {latestVersion !== '' && latestVersion !== version && (
+          {latestVersion !== '' && latestVersion !== version && isElectron && (
```

Comment:
This seems like it should still be just electron.

Reply:
yes, you can see I have a condition isElectron so it will only show the control if isElectron is true.

Reply:
oh...read that wrong that you were removing it...